docs/speech.rst

```diff
@@ -54,13 +45,24 @@ Functions
     that can be further hand-edited to improve accuracy, inflection and
     emphasis.
 
-.. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128)
+.. py:function:: pronounce(phonemes, \*, pitch=64, speed=72, mouth=128, throat=128, pin=microbit.pin_speaker)
 
     Pronounce the phonemes in the string ``phonemes``. See below for details of
     how to use phonemes to finely control the output of the speech synthesiser.
     Override the optional pitch, speed, mouth and throat settings to change the
     timbre (quality) of the voice.
 
+    As with the music module, you can use an optional argument to specify the 
+    output pin can be used to override the default of ``microbit.pin0``. 
+    If you have the latest micro:bit, you can use ``microbit.pin_speaker``
+    **V2**.
```

Comment:
It's not clear that this argument is v2 only.

Reply:
Updated this, but open to suggestions on how to make it more explicit

Reply:
Okay, I'll mark this comment as resolved and open a new one: https://github.com/bbcmicrobit/micropython/pull/692/files#r531537164